widget_driver_test/pubspec.yaml

```diff
@@ -3,14 +3,16 @@ description: Contains helper classes/methods for DrivableWidgets, WidgetDrivers,
 version: 0.0.5
 repository: https://github.com/bmw-tech/widget_driver/tree/master/widget_driver_test
 issue_tracker: https://github.com/bmw-tech/widget_driver/issues
+publish_to: "none"
```

Comment:
These changes are only tmp.
The widget_driver has a path dependency to the test package in its tests..
And the widget_driver_test has a real dependency to widget_driver..

Next PR will fix this once the new API of widget_driver is published